finsoc/app/assets/javascripts/newsDropdown.js.erb

```diff
@@ -0,0 +1,26 @@
+
+/* When the user clicks on the button, 
+toggle between hiding and showing the dropdown content */
+function selectNews() {
```

Comment:
Place the entire file in a closure. This will prevent function overrides just in case you name functions with the same title (essentially namespacing).